sections/discussion/ourprocessreflections.tex

```diff
@@ -54,6 +54,20 @@ \subsubsection{Formal reviews}
     \label{fig:formal-reviews}
 \end{figure}
 
+\subsubsection{Prototypes}
+In the beginning of the semester we created multiple prototypes for the entire system so that everyone had the same understanding of how the design should be.
+Some of these prototypes have been described in \autoref{sec:prototypes-design}.
+\\\\
+When the development of the prototypes started we had two people assigned to the task of creating high fidelity prototypes, and afterwards people reviewed the prototypes and gave feedback and asked for changes.
+Since the prototypes were high fidelity it was more difficult for reviewers to ask for big changes since it would take a lot of time to change.
+There was an initial discussion of the design, but as they were developed it was harder for all developers to give input for the design.
+A solution to this problem could be that the group developed low fidelity paper prototypes together. 
+This would give the people assigned the task of creating high fidelity prototypes something to develop them from, and would also lessen the amount of changes they had to do from the feedback.
+\\\\
+When we started developing the application the prototypes were used a lot.
+These were used for inspiration for every task, but later in the process they started to get used less often and lost a bit of their importance.
+A reason for this can also be that the developers were able to see the design pattern from other pages, which could give them a good idea of how to solve the tasks.
```

Comment:
```suggestion
A reason for this could be that the developers were able to see the design pattern from other implemented pages, which could give them a good idea of how to solve the tasks.
```